setup.py

```diff
@@ -54,7 +54,7 @@ def _dbt_athena_version() -> str:
     install_requires=[
         "dbt-core~=1.0.0",
```

Comment:
This should probably also be adjusted to `>=`or `~=1.0`

I would actually change all of these to `>=` and leave the responsibility to pick a working set o f packages to the end user. Given that the current set by the pinned versions also fails in some scenarios, it at least gives the user the flexibility to use something which works.

Reply:
Updated. Is it ok?